src/components/templates/AvatarGrid/RoomModal.tsx

```diff
@@ -67,18 +70,11 @@ export const RoomModal: React.FC<PropsType> = ({
     return <></>;
   }
 
-  const roomEvents =
-    venueEvents &&
-    venueEvents.filter(
-      (event) =>
-        event.room === room.title &&
-        event.start_utc_seconds +
-          event.duration_minutes * ONE_MINUTE_IN_SECONDS >
-          getCurrentTimeInUTCSeconds()
-    );
-
-  const usersInRoom = partygoers.filter(
-    (goer) => goer.room === `${venueName}/${room.title}`
+  const roomEvents = venueEvents.filter(
+    (event) =>
+      event.room === room.title &&
+      event.start_utc_seconds + event.duration_minutes * ONE_MINUTE_IN_SECONDS >
+        getCurrentTimeInUTCSeconds()
   );
```

Comment:
I'm pretty sure we can. I prefer having it done in batches so that we could easily review\merge the changes. Would be nice to have this separated into a hook in the following PRs. Will do useMemo in this one

Reply:
Will leave this unresolved for now in case we want to backlog something to extract this in future. Though I believe this specific file (all of AvatarGrid) will get removed in https://github.com/sparkletown/sparkle/pull/833 or similar